### ISSUE TITLE
gtk3: all themes is incorrectly rendered except Slider

### ISSUE THREAD

**Issue by rilian-la-te** [mate-desktop/mate-notification-daemon#32]
It is black or transparent. Screen:
![screenshot from 2014-05-04 21 29 18](https://cloud.githubusercontent.com/assets/1329638/2873206/84e5e7b6-d398-11e3-906f-74aa8fad20e1.png)

**Comment by infirit** [comment]
#68

**Comment by GiedriusS** [comment]
Still some issues with the newest git here. They render correctly but seems to have wrong size? For example, nodoka looks like this: 
https://i.imgur.com/C7rOYgj.jpg

**Comment by infirit** [comment]
It works, but for some reason not immediatly. It will get shaped when you hover over the close button for example. I have not figured out why this is :confused:. The code here, https://github.com/mate-desktop/mate-notification-daemon/blob/master/src/themes/nodoka/nodoka-theme.c#L509

**Comment by GiedriusS** [comment]
In themes/coco/coco-theme.c:paint_window() I've added printf to print out windata->width and windata->height. It gets called when I compile it with 2.0 and I get the output:
290 68
And when I add the same thing to themes/coco/coco-theme.c:on_configure_event() and it only gets called when I have compiled with gtk3 then I get:
290 200
So clearly something is wrong with the height. I'm not sure what calls those methods but I just wanted to get the width/height. Will look into the code more.
